src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany
     private List<Actor> actors;
 
     public Movie() {
```

Comment:
The default constructor is required by Hibernate, so it's good that you have it. However, it should be kept empty unless you have specific actions that need to be performed upon instantiation of the class by Hibernate.